### ISSUE TITLE
Release 3.1.0

### ISSUE THREAD

**Issue by er-vin** [nextcloud/desktop#2695]
### :memo: TO DO
- [x] :arrow_up: Bump VERSION.cmake to 3.1.0 in the master branch
- [x] :label: Create tag from master on the commit introducing the version bump
```
$ git checkout master
$ git pull
$ git tag -s -m "Release 3.1.0" v3.1.0 
$ git push origin --tags
```
- [x] :evergreen_tree: Create the new stable branch for the desktop repository
```
$ git checkout v3.1.0
$ git checkout -b stable-3.1
$ git push -u origin stable-3.1
```
- [x] :evergreen_tree: Create the new stable branch for the client-building repository
```
$ git checkout master
$ git checkout -b stable-3.1
$ git push -u origin stable-3.1
```
- [x] :arrow_up: Bump VERSION.cmake to 3.1.50 in the master branch
- :factory: Trigger the builds on brander and wait for them to succeed:
    - [x] :penguin: Linux
    - [x] :apple: Mac OS
    - [x] :door: Windows setup exe
    - [x] :door: Windows MSI
- :test_tube: Execute the feature release test plan for each build:
    - [x] :penguin: Linux
    - [x] :apple: Mac OS
    - [x] :door: Windows setup exe
    - [x] :door: Windows MSI
- [x] :book: Get changelog using [github_helper scripts](https://github.com/nextcloud/github_helper/), with previous feature release as origin point
- :book: Create draft pre-release at:
    - [x] [nextcloud/desktop/releases](https://github.com/nextcloud/desktop/releases) with the changelog
    - [x] [nextcloud-releases/desktop/releases](https://github.com/nextcloud-releases/desktop/releases)
- :package: Upload binaries to:
    - [x] the draft pre-release at [nextcloud/desktop/releases](https://github.com/nextcloud/desktop/releases)
    - [x] the draft pre-release at [nextcloud-releases/desktop/releases](https://github.com/nextcloud-releases/desktop/releases)
    - [x] [download.nextcloud.com/desktop/releases](https://download.nextcloud.com/desktop/releases)
- [x] :package: Add binaries to the draft releases (in both desktop repositories above)
- [x] :package: Copy above binaries to download.nextcloud.com/desktop/releases
- [x] :old_key: Notify Roeland that the binaries are available for signing
- [x] :bellhop_bell: Notify Marketing that the release is ready
- :rocket: When marketing gives the go then publish the draft pre-releases created previously at:
    - [x] [nextcloud/desktop/releases](https://github.com/nextcloud/desktop/releases)
    - [x] [nextcloud-releases/desktop/releases](https://github.com/nextcloud-releases/desktop/releases)
- [x] :up: Change the [updater server](https://github.com/nextcloud/client_updater_server/blob/master/config/config.php) config to serve the release

**Comment by Id3aFly** [comment]
I've seen that the 3.1.0 milestone still contains many open issues. Are they already solved or do you move the open issues to ~3.2?

**Comment by er-vin** [comment]
> I've seen that the 3.1.0 milestone still contains many open issues. Are they already solved or do you move the open issues to ~3.2?

Will be moved to 3.2. This was a way too optimistic plan and we had to deal with unexpected other issues during that cycle.
